about/media/index.md

```diff
@@ -10,30 +10,80 @@ categories: []
+    </div>
+    <a href="{% link about/media/ugrc_wavy_gravy_logos.zip %}" class="card__action text-center">{% include fa_icon.html download=true class="svg-inline--fa fa-w-16" %}
+    Download high resolution logos</a>
+  </div>
+  <div class="flex flex--col card">
+    <div class="flex card__content">
+      <img src="{% link about/media/ugrc-logo-SGID-wavy_gravy.png %}" alt="SGID logo" style="padding: 1em; align-self: center" loading="lazy" />
+    </div>
+    <a href="{% link about/media/ugrc_wavy_gravy_logos.zip %}" class="card__action text-center">{% include fa_icon.html download=true class="svg-inline--fa fa-w-16" %}
+    Download high resolution logos</a>
+  </div>
+  <div class="flex flex--col card">
+    <div class="flex card__content">
+      <img src="{% link about/media/ugrc-logo-SGID-dark.png %}" alt="SGID logo dark" style="padding: 1em; align-self: center" loading="lazy" />
+    </div>
+    <a href="{% link about/media/ugrc_wavy_gravy_logos.zip %}" class="card__action text-center">{% include fa_icon.html download=true class="svg-inline--fa fa-w-16" %}
+    Download high resolution logos</a>
+  </div>
+  <div class="flex flex--col card">
+    <div class="flex card__content" style="background-color:#1F0824">
+      <img src="{% link about/media/ugrc-logo-SGID-light.png %}" alt="SGID logo light" style="padding: 1em; align-self: center" loading="lazy" />
+    </div>
+    <a href="{% link about/media/ugrc_wavy_gravy_logos.zip %}" class="card__action text-center">{% include fa_icon.html download=true class="svg-inline--fa fa-w-16" %}
+    Download high resolution logos</a>
+  </div>
+  <div class="flex flex--col card">
+    <div class="flex card__content">
+      <img src="{% link about/media/ugrc-logo-Discover.png %}" alt="Discover base map logo" style="padding: 1em; align-self: center" loading="lazy" />
+    </div>
+    <a href="{% link about/media/ugrc_wavy_gravy_logos.zip %}" class="card__action text-center">{% include fa_icon.html download=true class="svg-inline--fa fa-w-16" %}
+    Download high resolution logos</a>
+  </div>
+  <div class="flex flex--col card">
+    <div class="flex card__content">
+      <img src="{% link about/media/ugrc-logo-TURN.png %}" alt="TURN GPS logo" style="padding: 1em; align-self: center" loading="lazy" />
+    </div>
+    <a href="{% link about/media/ugrc_wavy_gravy_logos.zip %}" class="card__action text-center">{% include fa_icon.html download=true class="svg-inline--fa fa-w-16" %}
+    Download high resolution logos</a>
   </div>
   <div class="flex flex--col card">
     <div class="flex card__content">
-      <img src="{% link about/media/agrc_horizontal.png %}" alt="Tiara Horizontal" style="padding: 1em; align-self: center" loading="lazy" />
+      <img src="{% link about/media/ugrc-logo-WebAPI.png %}" alt="UGRC Web API logo" style="padding: 1em; align-self: center" loading="lazy" />
```

Comment:
i like " let the data take you places" over " let data take you places".  but, I'm not stuck on it.  whatever you think.  that's my two cents.  i like the overall phrase.  nice work.

Reply:
I don't feel like changing it all for a the. I'm going to leave it for now.